### DISCUSSION TITLE
Not able to access Vercel from Vietnam

### DISCUSSION THREAD

**Discussion by Smilefounder** [vercel/community#1026] [Help] [answered]
We are not able to access to Vercel since this morning from Vietnam. Kindly support!

Best regards,
Huy Nguyen

**Comment by amyegan** [comment, marked as answer]
Thanks for letting us know @Smilefounder! The team is aware of the problem and working to solve it.

**Reply by Smilefounder** [reply]
Thanks for your support ❤️

**Comment by jamezduong** [comment]
any update?

**Comment by cdslxc** [comment]
I heard a rumour that Viettel ISP block Vercel like they block Behance & GoDadddy before.

**Comment by amyegan** [comment]
Thanks for reaching out about this, @jpduong and @cody-ta. The original issue was previously resolved, but it appears to be a problem again. The team is working with the ISP to unblock our IP addresses.

**Comment by Smilefounder** [comment]
Hi @amyegan I think the same issue is occur again since yesterday. Some of Vercel's IPs were not able to access from the Viettel ISP provider. 

Kindly review.

**Comment by dungnguyennvd** [comment]
Hi @amyegan @amyegan , this issues happen again. Some of Vercel's IPs were not able to access from the Viettel ISP provider
Please help me review. 
Thank you very much!

**Reply by amyegan** [reply]
Thanks for letting me know this is happening again. Our network team is aware of the problem and is working to solve it. 🙇

**Reply by dungnguyennvd** [reply]
Thanks for your support ❤️

**Comment by vhpx** [comment]
I'm having the same problem here, not with the website deployment but Vercel Edge Functions (via Next.js) always return 403 error when streaming data to any devices connected to a possibly Viettel-powered wifi in Vietnam. Using 4G/5G or VPN does make the issue goes away.

**Comment by dinhmai74** [comment]
Viettel may have blocked access to Vercel hosting again. My site got block too

**Comment by khiem20tc** [comment]
Viettel may have blocked access to Vercel hosting again. My site got blocked too today !!

**Reply by chrno1209** [reply]
I am having the same issue, @amyegan can you please take a look?

**Comment by dungnguyennvd** [comment]
Hi @amyegan , this issue is happen again, please help me check this. Thanks you very much!

**Comment by TasseDeCafe** [comment]
Can confirm, it's blocked in Vietnam.

**Comment by amyegan** [comment]
Thanks for reporting this. The team is looking into it 🙇

**Comment by jotaro-ethers** [comment]
Hi @amyegan @amyegan , this issues happen again. Some of Vercel's IPs were not able to access from the Viettel ISP provider
Please help me review.
Thank you very much!

**Reply by amyegan** [reply]
Thanks for letting me know. We reached out to Viettel to get these sites unblocked, but we don't have much control over when it will happen. Thank you for your patience while we work with them to get this solved. 🙇
